frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -62,7 +62,7 @@ You can see more information on [Operand's Creation Forms](../../descriptors#cre
 
 **x-descriptors**
 
-This descriptor allows you to specify the number of pods for your instance. See example from [[CSV] etcd Operator](https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/0.9.4/etcdoperator.v0.9.4.clusterserviceversion.yaml#L40-L44):
+This descriptor allows you to specify the number of pods for your instance. See example from [[CSV] etcd Operator](https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/0.9.4/etcdoperator.v0.9.4.clusterserviceversion.yaml#L48-L52):
```

Comment:
I'd suggest we link to this line in a specific commit so the link is stable in the future. (Here and the other links below.)

Reply:
The repository has been archived. I think it's safe to assume it won't change anytime soon, right?